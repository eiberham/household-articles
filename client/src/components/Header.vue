<template>
  <section class="header">
    <select>
      <option>Guardar y salir</option>
      <option>Salir sin guardar</option>
      <option>Guardar y continuar</option>
    </select>
  </section>
</template>

<script>
export default {
  /**
   * Reactive data.
   *
   * @returns {object}
   */
  data() {
    return {};
  }
};
</script>

<style lang="scss">
.header {
  background: #9a9996;
  color: #fff;
  grid: {
    area: header;
    column: 1 / -1;
  }

  height: 100px;

  display: flex;
  padding: 0 140px;
  justify-content: flex-end;
  align-items: center;

  &::before {
    background: {
      image: url("../assets/icons/home.svg");
      repeat: no-repeat;
      position: center center;
      size: 32px 32px;
    }
    content: "";
    display: block;
    position: absolute;
    top: 30px;
    left: 140px;
    height: 32px;
    width: 32px;
  }
}
</style>
