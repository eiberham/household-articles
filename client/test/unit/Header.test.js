import { mount } from "@vue/test-utils";
import Header from "../../src/components/Header.vue";

describe("Heeader", () => {
  const wrapper = mount(Header, {});

  test("is a Vue instance", () => {
    expect(wrapper.vm).toBeTruthy();
  });

  test("matches snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
