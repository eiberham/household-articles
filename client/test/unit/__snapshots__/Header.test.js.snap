// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heeader matches snapshot 1`] = `
<section
  class="header"
>
  <select>
    <option>
      Guardar y salir
    </option>
     
    <option>
      Salir sin guardar
    </option>
     
    <option>
      Guardar y continuar
    </option>
  </select>
</section>
`;
